tests/ga4gh/registry/test_server.py

```diff
@@ -29,9 +30,18 @@ def test_getServices():
     app.config['FOCA'] = Config(
         db=MongoConfig(**MONGO_CONFIG)
     )
+    mock_resp = deepcopy(MOCK_SERVICE)
+    mock_resp['id'] = MOCK_ID
+    app.config['FOCA'].db.dbs['serviceStore'].collections['services'] \
+        .client = mongomock.MongoClient().db.collection
+    app.config['FOCA'].db.dbs['serviceStore'].collections['services'] \
+        .client.insert_one(mock_resp)
+
+    data = deepcopy(MOCK_SERVICE)
+    data['id'] = MOCK_ID
```

Comment:
Might make sense to add another assert, where service list retrieval independent of ID is ensured?

Reply:
You could assert for list length of `1` in your case. You could also add a test where you _don't_ have any services added to check against the empty list. Or possibly you can add this assert to _this_ test by adding a `with app.app_context()` section before you call `.insert_one(mock_resp)`. But might still require to set a mock response (to `[]`), not sure.